Formatter/Pool.php

```diff
@@ -12,14 +12,16 @@
 namespace Sonata\FormatterBundle\Formatter;
 
 use Psr\Log\LoggerAwareInterface;
-use Psr\Log\LoggerInterface;
+use Psr\Log\LoggerAwareTrait;
 use Psr\Log\NullLogger;
 use Twig_Environment;
 use Twig_Error_Syntax;
 use Twig_Sandbox_SecurityError;
 
 class Pool implements LoggerAwareInterface
```

Comment:
We inject the logger ourselves, so I'd say no… but if we want the class to benefit from some automatic logger injection we're not aware about in some framework, this might be necessary. Doesn't hurt IMO. @sagikazarmark , what do you think?